### PR TITLE
pgwire: Handle LOGGER errors

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

**Comment by mfussenegger** [comment]
@aelesbao please review

File: sql/src/main/java/io/crate/protocols/postgres/ConnectionContext.java [hunk 1]
```diff
@@ -318,7 +318,15 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) throws
             try {
                 Messages.sendErrorResponse(ctx.getChannel(), e.getCause());
             } catch (Throwable t) {
-                LOGGER.error("Error trying to send error to client", t);
+                try {
+                    LOGGER.error("Error trying to send error to client", t);
```
Comment (aelesbao) [inline sql/src/main/java/io/crate/protocols/postgres/ConnectionContext.java]: why the logger was throwing here?

**Comment by aelesbao** [comment]
lgtm otherwise

File: sql/src/main/java/io/crate/protocols/postgres/ConnectionContext.java (hunk 1 above)
Comment (mfussenegger) [inline sql/src/main/java/io/crate/protocols/postgres/ConnectionContext.java]: Had some crazy logger initialization errors.

**Pull request merged**